projects/pgai/db/tests/vectorizer/test_named_vectorizer.py

```diff
@@ -0,0 +1,199 @@
+                ( id int not null generated always as identity
+                , title text not null
+                , published timestamptz
+                , body text not null
+                , drop_me text
+                , primary key (title, published)
+                )
+            """)
+            cur.execute(
+                """grant select, insert, update, delete on website.blog to bob, adelaide"""
+            )
+            cur.execute("""grant usage on schema website to adelaide""")
+            cur.execute("""
+                insert into website.blog(title, published, body)
+                values
+                  ('how to cook a hot dog', '2024-01-06'::timestamptz, 'put it on a hot grill')
+                , ('how to make a sandwich', '2023-01-06'::timestamptz, 'put a slice of meat between two pieces of bread')
+                , ('how to make stir fry', '2022-01-06'::timestamptz, 'pick up the phone and order takeout')
+            """)
+
+            # drop the drop_me column
+            cur.execute("alter table website.blog drop column drop_me")
+
+            # create a vectorizer for the blog table
+            # language=PostgreSQL
+            cur.execute("""
+            select ai.create_vectorizer
+            ( 'website.blog'::regclass
+            , name => 'blog_vectorizer'
+            , loading=>ai.loading_column('body')
+            , embedding=>ai.embedding_openai('text-embedding-3-small', 768)
+            , formatting=>ai.formatting_python_template('title: $title published: $published $chunk')
+            , scheduling=>ai.scheduling_timescaledb
+                    ( interval '5m'
+                    , initial_start=>'2050-01-06'::timestamptz
+                    , timezone=>'America/Chicago'
+                    )
+            , destination=>ai.destination_column('embedding1')
+            , chunking=>ai.chunking_none()
+            );
+            """)
+
+            cur.execute(
+                "select ai.vectorizer_queue_pending(name => %s)", ("blog_vectorizer",)
+            )
+            pending = cur.fetchone()[0]
+            assert pending == 3
+
+            # create another vectorizer with same name but if not exists set
+            cur.execute("""
+            select ai.create_vectorizer
+            ( 'website.blog'::regclass
+            , name => 'blog_vectorizer'
+            , loading=>ai.loading_column('body')
+            , embedding=>ai.embedding_openai('text-embedding-3-small', 768)
+            , formatting=>ai.formatting_python_template('title: $title published: $published $chunk')
+            , scheduling=>ai.scheduling_timescaledb
+                    ( interval '5m'
+                    , initial_start=>'2050-01-06'::timestamptz
+                    , timezone=>'America/Chicago'
+                    )
+            , destination=>ai.destination_column('embedding1')
+            , chunking=>ai.chunking_none()
+            , if_not_exists => true
```

Comment:
we need to test `if_not_exists=> true` with a destination_table as I think that's currently broken